app/ui/design-system/src/lib/Pages/InternalPage/index.tsx

```diff
@@ -96,10 +96,15 @@ export function InternalPage({
   const { pathname } = useLocation()
 
   useEffect(() => {
-    if (contentRef.current) {
+    if (
+      contentRef.current &&
+      !["cadence", "fcl-js"].includes(contentPath) &&
```

Comment:
Could we just check if the `header`  prop here is defined?

Reply:
Omg that's so much nicer.